debian/naemon-core.naemon.init

```diff
@@ -156,7 +156,12 @@ restart() {
 reload() {
     log_daemon_msg "Reloading $prog: "
     retval=0
-    pkill -HUP -u ${user} -f ${exec}
+    if [ -e $pidfile ]; then
+        PID=cat $pidfile;
```

Comment:
there should be backticks, right?

Reply:
Yes think so as well, i.e same as: https://github.com/naemon/naemon-core/commit/c523f9c1f2ca4ac0e4097a7c7fa7a5f1c5e23332